build-images/github-webhook-monitor/cmd/main.go

```diff
@@ -34,6 +34,7 @@ var latestDeliveryId string
 const (
 	latestIdPath                            = "/mnt/latestId"
 	SLEEP_TIME_SECONDS_BETWEEN_GITHUB_POLLS = 120
+	EVENT_ID_NOT_FOUND_ERROR_MSG            = "HTTP Status Code 404 - Event Not Found"
```

Comment:
This is supposed to be an error string ?
Shouldn't it explain more about what's happened ?
eg:
"Github can't find the event it previously told us about. It returned 404 when we tried to get more information about the event." or something ?

Reply:
Also, start the constant ERROR_MSG_xxxx as the most important thing is that it's an error message.

Reply:
Great that it's not just buried in the code as a literal though.